src/inc/quic_driver_helpers.h

```diff
@@ -139,6 +139,213 @@ class QuicDriverService {
         }
         return true;
     }
+
+    VOID DoStopSvc()
```

Comment:
Style in the function isn't consistent with our other code.

Reply:
Yeah I know, I plan on digging through that. Its just a straight copy right now of the sample code.